### DISCUSSION TITLE
Potential feedback on the sysinfo crate

### DISCUSSION THREAD

**Discussion by GuillaumeGomez** [nokyan/resources#360] [General]
Hi,

I was wondering if you gave a try to the [sysinfo](https://crates.io/crates/sysinfo) crate to query system information and more importantly: if you did try it, why didn't you use it in the end? In particular: is any feature you need missing?

Thanks in advance and congrats for your app, it looks nice. :)

**Comment by nokyan** [comment]
Hi!

I didn't know about your package until now. :)

> I was wondering if you gave a try to the [sysinfo](https://crates.io/crates/sysinfo) crate to query system information

Mainly because I kinda forgot looking if someone already made something like that... 😅 I just went ahead and did it myself
I'll give it a try and see if I can incorporate it into Resources, thanks for the heads-up. :)

**Reply by GuillaumeGomez** [reply]
No problem. I'm very interested into feedback if you give it a try to see if you encountered API issues, missing features or bugs. :)

Don't hesitate if you have questions too. ;)
